pages/demo.js

```diff
@@ -136,30 +102,31 @@ export default function Demo() {
     }, 100000)
   }
 
-  const mint = async (generatedWallet) => {
```

Comment:
the wallet is stored in the state so rather than pass it as a param I just used it from the state variable